pom.xml

```diff
@@ -288,7 +288,17 @@
                                     <regex>
                                         <pattern>com.github.checkstyle.regression.configuration.ConfigGenerator</pattern>
                                         <branchRate>80</branchRate>
-                                        <lineRate>91</lineRate>
+                                        <lineRate>92</lineRate>
+                                    </regex>
+                                    <regex>
+                                        <pattern>com.github.checkstyle.regression.extract.ExtractInfoProcessor</pattern>
+                                        <branchRate>100</branchRate>
+                                        <lineRate>66</lineRate>
+                                    </regex>
+                                    <regex>
+                                        <pattern>com.github.checkstyle.regression.extract.CheckstyleInjector</pattern>
+                                        <branchRate>0</branchRate>
+                                        <lineRate>0</lineRate>
```

Comment:
Please create issue to create some tests and coverage for this class at a later time.